.github/workflows/ci.yaml

```diff
@@ -256,11 +266,24 @@ jobs:
         uses: docker/build-push-action@v5
         with:
           context: .
-          file: .github/containers/test-installation/Dockerfile
+          file: .github/containers/test-installation/python-${{ matrix.python }}.Dockerfile
```

Comment:
In here you should probably pass an `arg:` to parameterize it.

Reply:
I use ` build-args` instead. Done